about.html

```diff
@@ -0,0 +1,121 @@
+                        show was first held in 2013, and since then, it has become one of the most anticipated events in
+                        the Pakistani entertainment calendar.
+                        The Hum Awards 2022 will take place in Toronto, Canada on Saturday, 24th of September 2022, in
+                        conjunction with Firstontario Center Hamilton. Only a few days remain before the 15th of
+                        September 2022 to vote for your favorite celebrity.
+                    </p>
+                </div>
+                <h2 class="text-lg mt-4 text-center hero-date font-light"> HUM Awards is amongst the top-rated awards
+                    shows in the South Asian sub-continent. for sponsership inquiries please contact
+                </h2>
+                <h3 class="font-bold mt-4">sales@humawards.ca
+                </h3>
+            </div>
+        </section>
+        <!-- logo section -->
+        <section class="logo py-16 flex flex-col justify-center items-center border border-zinc-200" id="logo">
+            <h2 class="text-center text-3xl logo-title pt-8">Hum Style Awards 2022 logo</h2>
+            <div class="line"></div>
+            <p class="mt-4 font-light">
+                This year's hum style awards logo was decided through the logo compitition and the logo shown below has
+                selected.
+            </p>
+            <div class="border border-zinc-200 mt-12 py-8 px-10">
+                <p class="font-bold text-2xl">Hum Style Awards</p>
+            </div>
+
+        </section>
+
+        <!-- past section -->
+
+        <section class="py-16 text-center" id="past">
+            <h2 class="text-center text-3xl logo-title pt-8">See the past Hum Style Awards</h2>
+            <div class="line"></div>
+            <p class="mt-8 font-light">Take a look at past events of Hum Style Awards at Houston, Texas and Toronto,
+                Canada. </p>
+            <div class="flex flex-col md:flex-row gap-x-2 gap-y-4 mt-12 mx-8 md:mx-6">
+                <div class="flex flex-col justify-center items-center past1 overlay-past">
+                    <h2 class="past-date text-4xl">2021</h2>
+                    <h3 class="past-desc font-bold">7th Hum Style Awards in Houston Texas</h3>
+                </div>
+                <div class="flex flex-col justify-center items-center past2 overlay-past">
+                    <h2 class="past-date text-4xl">2018</h2>
+                    <h3 class="past-desc font-bold">6th Hum Style Awards in Toronto Canada</h3>
+                </div>
+            </div>
+        </section>
+
+        <!-- footer -->
+
+        <footer class="footer border border-zinc-200 py-8 flex items-center flex-col justify-between">
+            <h3 class="text-xl font-bold">Hum Style Awards</h3>
+            <p class="font-light">Original Design by Cindy Shin in Behance</p>
+        </footer>
```

Comment:
1. - [ ] the footer should have a logo aligned to the left and text to the right , kindly try to add the text section to match the design 

![image](https://user-images.githubusercontent.com/89282221/190472585-beb56fda-8b86-4343-9f9d-dd952f7381c8.png)

-----

2. - [ ] the footer should have different background colors depending on the page and the media query, currently it has a white background for all pages and views, kindly fix it according to the design :

Mobile Size :

- should have a white background. 💹 (already white)
 
 Desktop Size :

- About Page: should have a dark background. ❌  (kindly fix this to be dark instead of white)
- Home Page: should have a white background.  💹 (already white)